packages/chain-adapters/src/dogecoin/DogecoinChainAdapter.ts

```diff
@@ -0,0 +1,458 @@
+    sendMax = false
+  }: GetFeeDataInput<KnownChainIds.DogecoinMainnet>): Promise<
+    FeeDataEstimate<KnownChainIds.DogecoinMainnet>
+  > {
+    const feeData = await this.providers.http.getNetworkFees()
+
+    if (!to || !value || !pubkey) throw new Error('to, from, value and xpub are required')
+    if (!feeData.data.average?.satsPerKiloByte || !feeData.data.slow?.satsPerKiloByte) {
+      throw new Error('undefined fee')
+    }
+
+    // fast returning -100000000?
+    if (!feeData.data.fast?.satsPerKiloByte || feeData.data.fast?.satsPerKiloByte < 0) {
+      feeData.data.fast = feeData.data.average
+    }
+    // We have to round because coinselect library uses sats per byte which cant be decimals
+    const fastPerByte = String(Math.round(feeData.data.fast.satsPerKiloByte / 1024))
+    const averagePerByte = String(Math.round(feeData.data.average.satsPerKiloByte / 1024))
+    const slowPerByte = String(Math.round(feeData.data.slow.satsPerKiloByte / 1024))
+
+    const { data: utxos } = await this.providers.http.getUtxos({
+      pubkey
+    })
+
+    const utxoSelectInput = {
+      to,
+      value,
+      opReturnData,
+      utxos,
+      sendMax
+    }
+
+    const { fee: fastFee } = utxoSelect({
+      ...utxoSelectInput,
+      satoshiPerByte: fastPerByte
+    })
+    const { fee: averageFee } = utxoSelect({
+      ...utxoSelectInput,
+      satoshiPerByte: averagePerByte
+    })
+    const { fee: slowFee } = utxoSelect({
+      ...utxoSelectInput,
+      satoshiPerByte: slowPerByte
+    })
+
+    return {
+      [FeeDataKey.Fast]: {
+        txFee: String(fastFee),
+        chainSpecific: {
+          satoshiPerByte: fastPerByte
+        }
+      },
+      [FeeDataKey.Average]: {
+        txFee: String(averageFee),
+        chainSpecific: {
+          satoshiPerByte: averagePerByte
+        }
+      },
+      [FeeDataKey.Slow]: {
+        txFee: String(slowFee),
+        chainSpecific: {
+          satoshiPerByte: slowPerByte
+        }
+      }
+    }
+  }
+
+  async getAddress({
+    wallet,
+    bip44Params = ChainAdapter.defaultBIP44Params,
+    accountType = ChainAdapter.defaultUtxoAccountType,
+    showOnDevice = false
+  }: bitcoin.GetAddressInput): Promise<string> {
+    if (!supportsBTC(wallet)) {
+      throw new Error('DogecoinChainAdapter: wallet does not support btc')
+    }
+
+    const { isChange } = bip44Params
+    let { index } = bip44Params
+
+    // If an index is not passed in, we want to use the newest unused change/receive indices
+    if (index === undefined) {
+      const { xpub } = await this.getPublicKey(wallet, bip44Params, accountType)
+      const account = await this.getAccount(xpub)
+      index = isChange
+        ? account.chainSpecific.nextChangeAddressIndex
+        : account.chainSpecific.nextReceiveAddressIndex
+    }
+
+    const path = toPath({ ...bip44Params, index })
+    const addressNList = path ? bip32ToAddressNList(path) : bip32ToAddressNList("m/84'/0'/0'/0/0")
```

Comment:
does this need a 3 in it?

Reply:
yep good catch. the default here should be `m/44'/3'/0'/0/0`. likely not appearing as an issue because we pass the path. will fix.